### PR TITLE
transaction-details: fix no paid boleto split rule

### LINKED ISSUE #1067: ReciepentList quebra a interface quando possui split rules
### Contexto
Fazendo alguns testes na página de detalhes de transações achei um erro quando o tenho uma transação em boleto ela explode erro de statusLegend.

🚨  Investigar causa🚨 

screen shot 2019-01-18 at 19 06 42
screen shot 2019-01-18 at 19 14 01

### Descrição técnica
Modelo de transação via boleto com split_rule:
```
{
    "object": "transaction",
    "status": "waiting_payment",
    "refuse_reason": null,
    "status_reason": "acquirer",
    "acquirer_response_code": null,
    "acquirer_name": "pagarme",
    "acquirer_id": "5a83ab2774e9653149459f31",
    "authorization_code": null,
    "soft_descriptor": null,
    "tid": 4824337,
    "nsu": 4824337,
    "date_created": "2019-01-18T18:24:54.755Z",
    "date_updated": "2019-01-18T18:24:54.962Z",
    "amount": 3980,
    "authorized_amount": 3980,
    "paid_amount": 0,
    "refunded_amount": 0,
    "installments": 1,
    "id": 4824337,
    "cost": 0,
    "card_holder_name": null,
    "card_last_digits": null,
    "card_first_digits": null,
    "card_brand": null,
    "card_pin_mode": null,
    "card_magstripe_fallback": false,
    "postback_url": null,
    "payment_method": "boleto",
    "capture_method": "ecommerce",
    "antifraud_score": null,
    "boleto_url": "https://pagar.me",
    "boleto_barcode": "1234 5678",
    "boleto_expiration_date": "2019-01-25T02:00:00.000Z",
    "referer": "api_key",
    "ip": "191.240.149.178",
    "subscription_id": null,
    "phone": null,
    "address": null,
    "customer": {
        "object": "customer",
        "id": 914737,
        "external_id": "#4321",
        "type": "individual",
        "country": "br",
        "document_number": null,
        "document_type": "cpf",
        "name": "Morpheus Fishburne",
        "email": "mopheus@nabucodonozor.com",
        "phone_numbers": [
            "+5511987654321",
            "+551132165498"
        ],
        "born_at": null,
        "birthday": "1965-12-22",
        "gender": null,
        "date_created": "2019-01-18T18:24:54.714Z",
        "documents": [
            {
                "object": "document",
                "id": "doc_cjr2dj002019ucl6e1tot55ii",
                "type": "cpf",
                "number": "98765432100"
            }
        ]
    },
    "billing": {
        "object": "billing",
        "id": 162953,
        "name": "Trinity Moss",
        "address": {
            "object": "address",
            "street": "Rua Matrix",
            "complementary": null,
            "street_number": "9999",
            "neighborhood": "Rio Cotia",
            "city": "Cotia",
            "state": "sp",
            "zipcode": "06714360",
            "country": "br",
            "id": 618527
        }
    },
    "shipping": null,
    "items": [
        {
            "object": "item",
            "id": "124",
            "title": "Blue pill",
            "unit_price": 1990,
            "quantity": 1,
            "category": null,
            "tangible": true,
            "venue": null,
            "date": null
        },
        {
            "object": "item",
            "id": "123",
            "title": "Red pill",
            "unit_price": 1990,
            "quantity": 1,
            "category": null,
            "tangible": true,
            "venue": null,
            "date": null
        }
    ],
    "card": null,
    "split_rules": [
        {
            "object": "split_rule",
            "id": "sr_cjr2dj01d019wcl6e4a9j7p72",
            "liable": false,
            "amount": null,
            "percentage": 35,
            "recipient_id": "re_cjq858fmw01b71w6d5gdunqx9",
            "charge_remainder": false,
            "charge_processing_fee": true,
            "block_id": null,
            "date_created": "2019-01-18T18:24:54.769Z",
            "date_updated": "2019-01-18T18:24:54.769Z"
        },
        {
            "object": "split_rule",
            "id": "sr_cjr2dj01c019vcl6e46hdml1h",
            "liable": true,
            "amount": null,
            "percentage": 65,
            "recipient_id": "re_cjq8580hx01lhvu6e4heix01n",
            "charge_remainder": true,
            "charge_processing_fee": true,
            "block_id": null,
            "date_created": "2019-01-18T18:24:54.769Z",
            "date_updated": "2019-01-18T18:24:54.769Z"
        }
    ],
    "metadata": {},
    "antifraud_metadata": {},
    "reference_key": null,
    "device": null,
    "local_transaction_id": null,
    "local_time": null,
    "fraud_covered": false,
    "order_id": null,
    "risk_level": "unknown",
    "receipt_url": null,
    "payment": null,
    "addition": null,
    "discount": null
}
```

### PR DESCRIPTION
**Pull request by alalbux**
## Contexto
Esse PR trata o caso de boleto não pago que possui split_rules de recebedores. 

## Checklist
- [x] Não renderizar componente em caso de boleto não pago

## Issues linkadas
- [x] Resolves #1067

## Como testar?

- Modelo de transação:
```json
{
  "api_key": "",
    "amount": 3980,
    "payment_method": "boleto",
    "customer": {
      "external_id": "#4321",
      "name": "Morpheus Fishburne",
      "type": "individual",
      "country": "br",
      "email": "mopheus@nabucodonozor.com",
      "documents": [
        {
          "type": "cpf",
          "number": "98765432100"
        }
      ],
      "phone_numbers": ["+5511987654321", "+551132165498"],
      "birthday": "1965-12-22"
    },
    "billing": {
      "name": "Trinity Moss",
      "address": {
        "country": "br",
        "state": "sp",
        "city": "Cotia",
        "neighborhood": "Rio Cotia",
        "street": "Rua Matrix",
        "street_number": "9999",
        "zipcode": "06714360"
      }
    },
    "items": [
      {
        "id": "123",
        "title": "Red pill",
        "unit_price": 1990,
        "quantity": 1,
        "tangible": true
      },
      {
        "id": "124",
        "title": "Blue pill",
        "unit_price": 1990,
        "quantity": 1,
        "tangible": true
      }
    ],
    "split_rules": [
      {
        "recipient_id": "-",
        "percentage": 65,
        "liable": true,
        "charge_processing_fee": true
      },
      {
        "recipient_id": "-",
        "percentage": 35,
        "liable": false,
        "charge_processing_fee": true
      }  
    ]
}
```

- `yarn start`

### CODE REVIEW

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: b9a117a51fb38493f715231298ca05f8b18f1a5d
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: b9a117a51fb38493f715231298ca05f8b18f1a5d
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: packages/pilot/src/containers/TransactionDetails/index.js [hunk 1]
```diff
@@ -724,7 +724,7 @@ class TransactionDetails extends Component {
             tv={9}
           >
             <Grid>
-              {!isEmptyOrNull(recipients) &&
+              {!isEmptyOrNull(recipients) && !isBoletoWaitingPayment(transaction) &&
```
Comment (uselessdev) [inline packages/pilot/src/containers/TransactionDetails/index.js]: De onde ver essa função `isBoletoWaitingPayment `?

Comment (Deivis) [reply packages/pilot/src/containers/TransactionDetails/index.js]: daqui https://github.com/pagarme/pilot/blob/83fc077067f77ef4919db200a095aaae74bf04af/packages/pilot/src/containers/TransactionDetails/index.js#L107 @uselessdev

**Pull request merged**